### DISCUSSION TITLE
Buzzcaptions mac store app

### DISCUSSION THREAD

**Discussion by DeafNet** [chidiwilliams/buzz#575] [General]
Is there supposed to auto saved to the file after it completes the audio ? Because I had to do the manual and it is not the same as window .

**Comment by chidiwilliams** [comment]
Yes, Buzz Captions doesn’t auto export atm, but I can make a change to add that it that’s something you want.

**Reply by DeafNet** [reply]
I kindly request your consideration in making a remarkable adjustment. While working on American Sign Language projects, I face challenges due to being Deaf. The implementation of this program would not only save time but also lead to cost savings. This software could operate while I rest or work, generating income to meet my financial responsibilities. It’s worth noting that my efforts in this endeavor are unpaid.

For those interested in experiencing the music through Braille, I utilize closed captions on YouTube for accessibility. You can find these at the link “YouTube.com/danielmeekasl”.

After transcribing, I meticulously analyze each musical phrase to ensure a seamless synchronization of the spoken parts. Once completed, the transcription becomes an Srt file for closed captioning and then transforms into an Lrc format for karaoke lyrics. This setup helps during performances with teleprompter equipment, allowing both vocal and visual engagement with the camera.

I sincerely appreciate the time and effort you’ve dedicated to this matter. Your kind

**Comment by DeafNet** [comment]
> Yes, Buzz Captions doesn’t auto export atm, but I can make a change to add that it that’s something you want.
> 
> 
> 
> > On 19 Aug 2023, at 13:22, DeafNet ***@***.***> wrote:
> 
> > 
> 
> > 
> 
> > Is there supposed to auto saved to the file after it completes the audio ? Because I had to do the manual and it is not the same as window .
> 
> > 
> 
> > —
> 
> > Reply to this email directly, view it on GitHub <https://github.com/chidiwilliams/buzz/discussions/575>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AEINE6CR43O5C4JQZSMAYRLXWCVZNANCNFSM6AAAAAA3WRMIZI>.
> 
> > You are receiving this because you are subscribed to this thread.
> 
> > 
> 
> 
> 
> 

Will this will be in the next update on Mac store ? Because it will be huge help.
